oem/ibm/libpldmresponder/file_io_type_lid.hpp

```diff
@@ -150,10 +155,12 @@ class LidHandler : public FileHandler
             {
                 if (strcmp(e.name(), accessKeyExpired) != 0)
                 {
+                    phosphor::logging::commit<AccessKeyExpired>();
                     validateStatus = ENTITLEMENT_FAIL;
                 }
                 else if (strcmp(e.name(), incompatibleErr) != 0)
```

Comment:
The check should be for a return value of 0:
if (strcmp(e.name(), accessKeyExpired) == 0)

strcmp() returns 0 if the contents of both strings are equal.

Reply:
Done